index.html

```diff
@@ -1002,7 +1002,7 @@ <h3>Workflows and Exchanges</h3>
 addition to the VC API exchange protocol; for example, an exchange could
 potentially be executable with any of the OID4VCI, OID4VP, DIDComm, and
 VC API exchange protocols. Which protocols are supported by a given workflow
```

Comment:
Successive re-reads make me think this needs a larger revision. This one must be taken with the one below, or both must be rejected.
```suggestion
VC API exchange protocols. The protocols supported by a given workflow depend
on the implementation of the workflow service, the complexity of the workflow
```